qutebrowser/completion/models/sortfilter.py

```diff
@@ -69,6 +71,10 @@ def set_pattern(self, val):
         """
         with debug.log_time(log.completion, 'Setting filter pattern'):
             self.pattern = val
+            val = val.casefold()
```

Comment:
I guess since we're using regexes now, we could simply use `re.IGNORECASE` when compiling instead of using `casefold()`.